### ISSUE TITLE
Difference between model_s012_final_yv and model_s012_final

### ISSUE THREAD

**Issue by bo-miao** [Limingxing00/RDE-VOS-CVPR2022#1]
Hi,

Thanks for releasing the code. May I know the difference between model_s012_final_yv and model_s012_final?

**Comment by Limingxing00** [comment]
model_s012_final is used for davis 16 val, davis 17 val and davis 17 test.
model_s012_final_yv is used for youtube 19.

The only difference is that in the loss function (s012 stage), we reduce the weights of lmc by half.   

Reason:
If the same model of davis is used, j&f is 0.830, which is still a superior result. But lowering lmc reduces the risk of overfitting the seen category on youtube-vos.

**Comment by bo-miao** [comment]
Thanks for the answer!
